Marlin/example_configurations/delta/FLSUN/auto_calibrate/Configuration.h

```diff
@@ -450,8 +450,8 @@
   // See http://minow.blogspot.com/index.html#4918805519571907051
   #define DELTA_CALIBRATION_MENU
 
-  // set the radius for the calibration probe points - max 0.8 * DELTA_PRINTABLE_RADIUS if DELTA_AUTO_CALIBRATION enabled
-  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS - 17) // mm
+  // set the radius for the calibration probe points - max DELTA_PRINTABLE_RADIUS*0.869 if DELTA_AUTO_CALIBRATION enabled
+  #define DELTA_CALIBRATION_RADIUS (DELTA_PRINTABLE_RADIUS*0.869) // mm
```

Comment:
PS: I've learned from reading (and copying) the existing code and I've noticed that  `DEFINED_THING` was never set to something like `blablabla`, but always to `(blablabla)`, so this is not a critical mistake. But since Marlin is appently in a transitional state from putting the parenthesis's around `blablabla` to putting them around `DEFINED_THING`, I'll put them around both.